### ISSUE TITLE
How to label my own image

### ISSUE THREAD

**Issue by huidemie** [CoinCheung/DeepLabv3plus#14]
Thank you for your coding. I want to use my own images to train the model. So when I label my images, I just need to label them like "xxxxgtFine_labelIds.png " .Other file such as ".json" and other ".png" file is not needed, right?

**Comment by VipulRamtekkar** [comment]
Yup, if you keep the same format for labelling as cityscapes that should be fine.
